sample-lnd.conf

```diff
@@ -45,12 +53,15 @@
 
 ; Adds an extra domain to the generate certificate. Setting multiple tlsextradomain= entries is allowed.
 ; (old tls files must be deleted if changed)
-; tlsextradomain=
+; Default:
+;   tlsextradomain=
+; Expample (option can be specified multiple times):
+;   tlsextradomain=my-node-domain.com
 
 ; If set, then all certs will automatically be refreshed if they're close to
 ; expiring, or if any parameters related to extra IPs or domains in the cert
 ; change.
-; tlsautorefresh=true
+; tlsautorefresh=false
```

Comment:
Just a note, I think these booleans are set to `true` for convenience so that removing the `;` is equivalent to enabling the flag. If the default is 'false' then why would the node operator uncomment the line for reasons other than enabling?

Reply:
We had short discussion about the booleans here
https://github.com/lightningnetwork/lnd/issues/7612#issuecomment-1519693477

The idea was to reflect always the default value for each option in sample-lnd.conf. To be consistent for booleans the default is always false. 
Yes, you have to change the value when you uncomment the line. But that's the case also for the other options now.